### DISCUSSION TITLE
1C/1C showing higher cycle life than 0.5C/0.5C

### DISCUSSION THREAD

**Discussion by sths008** [pybamm-team/PyBaMM#4115] [Q&A]
I am running 1C/1C charge-discharge and 0.5C/0.5C using various lithium ion models. All models are showing 1C/1C with higher cycling life than 0.5C/0.5C. Here is an example of what I am doing.

 
import pybamm as pb
param = pb.ParameterValues("OKane2022")

model = pb.lithium_ion.DFN(
    {
        "SEI": "ec reaction limited",
        "SEI film resistance": "distributed",
        "SEI porosity change": "true",
        "lithium plating": "irreversible",
        "lithium plating porosity change": "true",
    }
)

experiment = pb.Experiment(
    [
        (
            "Discharge at 1 C until 2.85 V",
            "Rest for 5 minutes",
            "Charge at 1 C until 4.15 V",
            "Hold at 4.15 V until C/10",
            "Rest for 5 minutes",
            
        )
    ]
    * 500,termination = "90% capacity"
    
)

solver = pb.CasadiSolver(
    mode="safe",
    dt_max=0.01, 
    max_step_decrease_count=15,  
    rtol=1e-6,  
    atol=1e-6,  
    extra_options_setup={"max_num_steps": 10000, "print_stats": False}, 
    perturb_algebraic_initial_conditions=True,  
)

sim = pb.Simulation(model, experiment=experiment, parameter_values=param, solver=solver)
sim.solve()

I have uninstalled pybamm and reinstalled it but I am still getting this wrong results. I am using version 24.1.

I would appreciate if someone could help me with this issue.

**Comment by valentinsulzer** [comment]
Depending on the SEI model you use calendar aging may be dominating in which case higher C-rate means less time per cycle. It looks like slower degradation when plotted vs cycle number but is faster (or the same) when plotted vs time. See https://github.com/pybamm-team/PyBaMM/discussions/3220

Additionally, temperature plays an important role, so try with the thermal model activated. Higher C-rate means higher temperature so faster SEI growth.
